html/pfappserver/lib/pfappserver/I18N/en.po

```diff
@@ -10312,7 +10312,7 @@ msgstr ""
 
 # pf::action (VIOLATION_ACTIONS)
 msgid "log_action"
-msgstr "Log message"
+msgstr "Log violation"
```

Comment:
Or "Log violation message" ?

Reply:
IMO "message" seems excessive and is implied with "Log".